Backpack/Dialog/Classes/BPKDialogIconView.m

```diff
@@ -49,18 +49,18 @@ - (CGSize)intrinsicContentSize {
     return [[self class] viewSize];
 }
 
-- (void)setIconBackgroundColor:(UIColor *_Nullable)iconBackgroundColor {
-    self.iconContainerView.backgroundColor = iconBackgroundColor;
+- (void)setIconDefinition:(BPKDialogIconDefinition *_Nullable)iconDefinition {
+    // TODO handle iconDefinition nil
```

Comment:
This still isn't done

Reply:
Shouldn't need to handle `nil` case as the `iconView` should always have an `iconDefinition`!